t/expected_test_results/attributes/en-attributes.json

```diff
@@ -295,7 +295,7 @@
                "grade" : "b",
                "icon_url" : "https://server_domain/images/attributes/ecoscore-b.svg",
                "id" : "ecoscore",
-               "match" : 61,
+               "match" : 63,
```

Comment:
How is this that you added 2, while the code added 1 ?

Reply:
In fact previously we removed 1, and now we add 1, hence the difference of 2 in the tests.

Reply:
Oh yes, sorry !